Documentation/guides/extract-native-libraries.md

```diff
@@ -42,8 +42,9 @@ Xamarin.Android build system will do the following:
     extension, causing native libraries to be stored uncompressed
     within the `.apk`.
 
- 2. The `__XA_DSO_IN_APK` environment variable will be set within the
-    created `.apk` file with the value of `1`, indicating to
-    the app that native libraries should be loaded from the `.apk`
-    itself instead of from the filesystem.
+Previously, a `__XA_DSO_IN_APK` environment variable was set, but this
```

Comment:
Do we need this information? Was this something that the user was able to set manually?